Python/Tests/Utilities.Python/PythonReplWindowProxySettings.cs

```diff
@@ -78,12 +78,22 @@ public sealed class PythonReplWindowProxySettings : ReplWindowProxySettings {
                 interpreters.FindConfiguration(Version.Id)
             );
 
+            if (!string.IsNullOrEmpty(projectName)) {
+                var dteProj = app.GetProject(projectName);
+                var proj = (PythonProjectNode)dteProj.GetCommonProject();
+                replId = PythonReplEvaluatorProvider.GetEvaluatorId(proj);
+            }
+
             return app.ServiceProvider.GetUIThread().Invoke(() => {
+                app.ServiceProvider.GetPythonToolsService().InteractiveBackendOverride = executionMode == "IPython" ? IPythonBackend : StandardBackend;
```

Comment:
Can we just use the backend value as the `executionMode` parameter? As we add more backends (e.g. #2722) it will make it easier to test them.